corehq/apps/reports/templates/reports/form/edit_submission.html

```diff
@@ -24,11 +24,11 @@
 {% endblock %}
 
 {% block title %}{% trans 'Edit Submission' %}{% endblock %}
-{% block page-title %}
-    {% include 'reports/form/partials/form_breadcrumbs.html' %}
+{% block page_title %}
+    {% include 'reports/form/partials/bootstrap3/form_breadcrumbs.html' %}
 {% endblock %}
 
-{% block main_column %}
+{% block page_content %}
     <div class="inline-touchforms" data-bind="template: { name: 'form-fullform-ko-template' }" id="edit-container"></div>
     <div id="loading" class="hq-loading" style="display: none;">
         <img width="32" height="32" alt="Loading" src="{% static 'formplayer/img/loading_transparent.gif' %}" />
```

Comment:
will update (this won't break staging/prod since it doesn't end in `js`, `css`, or `less`